apps/connect/src/hooks/useFormatAssetParam.test.ts

```diff
@@ -3,19 +3,19 @@ import { useFormatAssetParam } from "./useFormatAssetParam";
 
 describe("useFormatAssetParam", () => {
   it("should get a formatted asset when key has an exact match", () => {
-    const { result } = renderHook(() => useFormatAssetParam("ETH"));
-    expect(result.current).toEqual("ETH");
+    const { result } = renderHook(() => useFormatAssetParam("BNB"));
```

Comment:
is there any reason to change from ETH to BNB?

Reply:
For some reason, it gives errors in the tests

<img width="758" alt="image" src="https://github.com/user-attachments/assets/c91ba154-4ed4-47fb-9877-4aaad6cf51fb">

Reply:
In the array that is supposed to be the ETH info, here is the result:
<img width="610" alt="image" src="https://github.com/user-attachments/assets/b8c789d8-9c6d-451f-92e7-f32f74eef2be">

Reply:
I think this is fixed